internal/uvm/scsi/mount.go

```diff
@@ -39,6 +39,7 @@ type mount struct {
 // MountConfig specifies the options to apply for mounting a SCSI device in
 // the guest OS.
 type MountConfig struct {
+	Partition uint64
```

Comment:
We need to add a check below in `mountConfigEquals` for the partition

Reply:
Now using `reflect.DeepEqual`.